app/code/Edu/StoreFinder/Block/Storefinder.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * @category Edu
+ * @package Edu\Ultimate
+ * @author Maxim Dzyuba <maxim.d@tdo.kz>
+ */
+
+namespace Edu\StoreFinder\Block;
+
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\View\Element\Html\Select;
+use Magento\Framework\View\Element\Template\Context;
+use Scandiweb\StoreFinder\Block\Storefinder as SF;
+use Scandiweb\StoreFinder\Helper\Data as DataHelper;
+
+class Storefinder extends SF
```

Comment:
Please use more descriptive class names than `SF`. Could call it CoreStoreFinder or something